src/series/Pokemon.yml

```diff
@@ -110,71 +106,70 @@ games:
         platforms: []
         release_year: null
         sub-series: Core
-    (III-V) Gen III secondary & remakes:
+    (III) Gen III secondary & remakes:
```

Comment:
these errors come from parsing the original markdown from the old website, we were parsing text such as

`[(III-V) Gen III secondary & remakes / Gen IV secondary & remakes / Pokémon White Version 2](https://warprandomizer.com/) (Universal Warp Randomizer) - [Source code](https://warprandomizer.com//randomizer_py.zip)`